Explorer/Assets/DCL/SDKComponents/TextShape/TMPProSdkExtensions.cs

```diff
@@ -54,25 +54,27 @@ public static void Apply(this TextMeshPro tmpText, PBTextShape textShape, IFonts
             tmpText.maxVisibleLines = textShape.HasLineCount && textShape.LineCount != 0 ? Mathf.Max(textShape.LineCount, 1) : int.MaxValue;
             tmpText.enableWordWrapping = textShape is { HasTextWrapping: true, TextWrapping: true } && !tmpText.enableAutoSizing;
 
-            tmpText.renderer.SetPropertyBlock(
-                materialPropertyBlock.Prepare(textShape));
-        }
-
-        private static MaterialPropertyBlock Prepare(this MaterialPropertyBlock materialPropertyBlock, PBTextShape textShape)
-        {
-            materialPropertyBlock.Clear();
-
-            // TODO (Vit) : Disable outline when if (textShape.OutlineWidth <= 0)
-            materialPropertyBlock.SetColor(ID_OUTLINE_COLOR, textShape.OutlineColor?.ToUnityColor() ?? Color.white);
-            materialPropertyBlock.SetFloat(ID_OUTLINE_WIDTH, textShape.OutlineWidth);
+            tmpText.renderer.GetPropertyBlock(materialPropertyBlock);
 
-            // TODO (Vit ): disable shadow when if (textShape.ShadowOffsetX == 0 || textShape.ShadowOffsetY == 0)
-            materialPropertyBlock.SetColor(ID_UNDERLAY_COLOR, textShape.ShadowColor?.ToUnityColor() ?? Color.white);
-            materialPropertyBlock.SetFloat(ID_UNDERLAY_SOFTNESS, textShape.ShadowBlur);
-            materialPropertyBlock.SetFloat(ID_UNDERLAY_OFFSET_X, textShape.ShadowOffsetX);
-            materialPropertyBlock.SetFloat(ID_UNDERLAY_OFFSET_Y, textShape.ShadowOffsetY);
```

Comment:
We're missing the X and Y shadow offset, is there a reason to remove it?

Reply:
You're are completely right! Trying things and different aaproaches in the code I forgot to keep these 2 properties. I've added them back again: https://github.com/decentraland/unity-explorer/pull/2007/commits/783ee9a1fecba5f64c505ea692826cb411701cf9